common/primitives/src/schema.rs

```diff
@@ -13,7 +13,7 @@ pub type SchemaId = u16;
 
 /// Types of modeling in which a message payload may be defined
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-#[derive(Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq, MaxEncodedLen)]
+#[derive(Copy, Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq, MaxEncodedLen)]
```

Comment:
Copy means that we don't have to worry about passing by reference as much in this case

Reply:
Is it safer approach for our inbuilt types then would be good to copy on passing